R/tags-diff.R

```diff
@@ -50,7 +50,7 @@ tags_diff <- function(.bbi_object, .mod2 = NULL, .print = TRUE, ...) {
 #' @export
 tags_diff.bbi_model <- function(.bbi_object, .mod2 = NULL, .print = TRUE, ...) {
   # check that it's a model and not a summary, etc.
-  check_model_object(.bbi_object)
+  check_yaml_in_sync(.bbi_object)
```

Comment:
this is unrelated to this PR, but I realized when writing `model_diff()` that I should check this here too.